src/bin/ion/commands/jq.rs

```diff
@@ -506,7 +466,7 @@ impl Neg for JaqElement {
             // Only number types can be negated
             Int(a) => (-a).into(),
             Float(a) => (-a).into(),
-            Decimal(a) => (-a.to_big_decimal()).to_decimal().into(),
+            Decimal(a) => (-a.into_big_decimal()).into_decimal().into(),
```

Comment:
I tried doing this with native logic built around Decimal's components, but it ended up inflating into more code than I wanted to own. If we implement any support in Decimal directly that would be a different story.

Reply:
You should open an issue for this in ion-rust. It's certainly not a blocker for this PR, but this did expose a rough edge in the API that we can pick up sometime.